doc/nrf/protocols/matter/end_product/security.rst

```diff
@@ -161,6 +161,80 @@ This is a reference configuration that can be modified in the production firmwar
        To use the Oberon backend for specific cryptographic operations supported by both drivers, disable those operations in the CRACEN driver, as it takes priority when both are enabled.
        See the :ref:`nrf_security_drivers` documentation for more information.
 
+.. _matter_platforms_security_kmu:
+
+nRF54L15 Key Management Unit (KMU)
+==================================
+
+nRF54l15 devices include :ref:`ug_nrf54l_crypto_kmu_cracen_peripherals` that can be used to store cryptographic keys in Matter.
+In this solution, the keys are stored within the available slots in the :ref:`ug_nrf54l_crypto_kmu_slots` range that are not reserved for current and future |NCS| use cases.
+
+The default slots range used for Matter is from ``100`` to ``180``, excluding the DAC private key.
+For details on the DAC private key configuration, see :ref:`matter_platforms_security_dac_priv_key_kmu`.
+To change the slots range, set the :kconfig:option:`CONFIG_CHIP_KMU_SLOT_RANGE_START` and :kconfig:option:`CONFIG_CHIP_KMU_SLOT_RANGE_END` Kconfig options.
+Currently, the Raw usage scheme defined in the :ref:`ug_nrf54l_crypto_kmu_key_usage_schemes` section is used.
+
+To use this feature, set the :kconfig:option:`CONFIG_CHIP_STORE_KEYS_IN_KMU` Kconfig option to ``y``, and switch to the ``KMUKeyAllocator`` by calling the ``chip::Crypto::SetPSAKeyAllocator`` method in your code during the Matter stack initialization.
+
+For example:
+
+.. code-block:: cpp
+
+    #include <platform/nrfconnect/KMUKeyAllocator.h>
+
+    static KMUKeyAllocator kmuAllocator;
+    Crypto::SetPSAKeyAllocator(&kmuAllocator);
+
+See the :file:`samples/matter/common/src/app/matter_init.cpp` file for a usage example.
+
+Due to limited slots available in the KMU, the maximum number of Matter fabric is limited.
+The following table shows the current number of slots used by Matter:
+
+.. list-table:: KMU slots used by Matter crypto materials
+   :widths: auto
+   :header-rows: 1
+
+   * - Crypto material in Matter
+     - Key type
+     - Number of slots needed for a key
+     - Multiplication
+     - Minimum number of keys
+   * - Node Operational Certificate (NOC) private key
+     - ECC secp256r1 key pair
+     - 2
+     - For each Matter fabric
+     - 5 (5 Matter fabrics for each device)
+   * - Intermittently Connected Device (ICD) Token
+     - HMAC SHA-256 128-bit keys
+     - 1
+     - For each ICD user
+     - 10 (2 ICD users for each Matter fabric)
+   * - Intermittently Connected Device (ICD) symmetric Key
+     - AES 128-bit keys
+     - 1
+     - For each ICD user
+     - 10 (2 ICD users for each Matter fabric)
+   * - Device Attestation Certificate (DAC) private key
+     - ECC secp256r1 key pair
+     - 2
+     - For each device
+     - 1
+   * - Group key [3]_
+     - AES 128-bit keys
+     - 1
+     - For each group
+     - 15 (3 groups for each Matter fabric)
+
```

Comment:
I know that DAC is not part of Matter Fab Credentials - but i would like to add DAC here as well and provide the default KEY_ID.

Reply:
We already have the DAC private key in the table.